packages/fxa-payments-server/src/components/Header/index.tsx

```diff
@@ -32,7 +32,7 @@ export const Header = ({ profile, className = 'default' }: HeaderProps) => {
     >
       <div data-testid="branding">
         <img
-          src={firefoxLogo}
+          src={mozillaLogo}
           alt={l10n.getString('brand-name-firefox-logo', null, 'Firefox logo')}
```

Comment:
Update to `brand-name-mozilla-logo` and `Mozilla logo`.

Additionally, `brand-name-firefox-logo` can be removed from `/Users/msardesai/fxa/packages/fxa-payments-server/src/components/Header/en.ftl` as it is no longer being used anywhere.

Reply:
Good catch. Thank you!

Reply:
BTW, we have a follow up task for removing old l10n strings.  I've left a comment in Header/en.ftl as a reminder.